script.js

```diff
@@ -1,9 +0,0 @@
-function add(a, b) {
```

Comment:
I know you said this was used later in the course. I really don't like the code from the Udemy course. Great code is self-documenting